Makefile

```diff
@@ -1,31 +1,27 @@
-.PHONY: run clean
+.PHONY: all
 
 SUDO=$(shell docker info >/dev/null 2>&1 || echo "sudo -E")
-EXE=iops
-ORGANIZATION=weaveworksplugins
+EXE=scope-plugin
+ORGANIZATION=openebs
 IMAGE=$(ORGANIZATION)/scope-$(EXE)
 NAME=$(ORGANIZATION)-scope-$(EXE)
 UPTODATE=.$(EXE).uptodate
 
-run: $(UPTODATE)
-	# --net=host gives us the remote hostname, in case we're being launched against a non-local docker host.
-	# We could also pass in the `-hostname=foo` flag, but that doesn't work against a remote docker host.
-	$(SUDO) docker run --rm -it \
-		--net=host \
-		-v /var/run/scope/plugins:/var/run/scope/plugins \
-		--name $(NAME) $(IMAGE)
+all: run build 
 
+run:
+	go build -v
+
+build: $(UPTODATE)
```

Comment:
This can not be `build`.
Refer: https://github.com/openebs/maya/blob/master/GNUmakefile#L247-L255